sdk/storage/Azure.Storage.Files.Shares/src/ShareDirectoryClient.cs

```diff
@@ -1785,7 +1785,66 @@ await GetSubdirectoryClient(subdirectoryName)
         /// </summary>
         /// <remarks>
         /// This method only initializes the file.
-        /// To add content, use <see cref="ShareFileClient.UploadRangeAsync"/>.
+        /// To add content, use <see cref="ShareFileClient.UploadRange(ShareFileRangeWriteType, HttpRange, System.IO.Stream, byte[], IProgress{long}, FileRequestConditions, CancellationToken)"/>.
+        /// </remarks>
+        /// <param name="fileName">The name of the file.</param>
+        /// <param name="maxSize">
+        /// Required. Specifies the maximum size for the file.
+        /// </param>
+        /// <param name="httpHeaders">
+        /// Optional standard HTTP header properties that can be set for the file.
+        /// </param>
+        /// <param name="metadata">
+        /// Optional custom metadata to set for the file.
+        /// </param>
+        /// <param name="smbProperties">
+        /// Optional SMB properties to set for the file.
+        /// </param>
+        /// <param name="filePermission">
+        /// Optional file permission to set for the file.
+        /// </param>
+        /// <param name="cancellationToken">
+        /// Optional <see cref="CancellationToken"/> to propagate
+        /// notifications that the operation should be cancelled.
+        /// </param>
+        /// <returns>
+        /// A <see cref="Response{FileClient}"/> referencing the file.
+        /// </returns>
+        /// <remarks>
+        /// A <see cref="RequestFailedException"/> will be thrown if
+        /// a failure occurs.
+        /// </remarks>
+        [ForwardsClientCalls]
+        [System.Diagnostics.CodeAnalysis.SuppressMessage("Usage", "AZC0002:DO ensure all service methods, both asynchronous and synchronous, take an optional CancellationToken parameter called cancellationToken.", Justification = "<Pending>")]
+        public virtual Response<ShareFileClient> CreateFile(
```

Comment:
We should add `[EditorBrowsable(false)]` to these overloads so they don't confuse customers.

Reply:
Argument 1: Cannot convert from bool to System.CompontentModel.EditorBrowserState.

Reply:
Try EditorBrowsableState.Never

Reply:
fixed.